examples/ex39p.cpp

```diff
@@ -0,0 +1,317 @@
+//                       MFEM Example 39 - Parallel Version
+//
+// Compile with: make ex39p
+//
+// Sample runs:  mpirun -np 4 ex39p
+//               mpirun -np 4 ex39p -ess "Southern Boundary"
+//               mpirun -np 4 ex39p -src Base
+//
+// Device sample runs:
+//               mpirun -np 4 ex39p -fa -d cuda
```

Comment:
Uncovered by the nightly tests: this sample run is not valid -- the option `-fa` is not supported at the moment.